src/styles/index.ts

```diff
@@ -1622,9 +1622,9 @@ const styles = (theme: ThemeColors) =>
                 ...getPopOverVerticalOffset(202 + 40),
             } satisfies AnchorPosition),
 
-        createMenuPositionReportActionCompose: (windowHeight: number) =>
+        createMenuPositionReportActionCompose: (shouldUseNarrowLayout: boolean, windowHeight: number, windowWidth: number) =>
             ({
-                horizontal: 18 + variables.sideBarWidth,
+                horizontal: shouldUseNarrowLayout ? windowWidth - (variables.sideBarWidth - 18) : 18 + variables.sideBarWidth,
```

Comment:
do you mind extracting `windowWidth - (variables.sideBarWidth - 18)` to a separate variable to give it a meaningful name? To me it's not obvious why this is counted differently for `shouldUseNarrowLayout`